docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -208,6 +208,10 @@ PowerShell module options and option choices are currently case insensitive to w
 specification. This behaviour is deprecated and a warning displayed to the user if a case insensitive match was found.
 A future release of Ansible will make these checks case sensitive.
 
+The file module defaults to ``recurse=no`` but didn't respect that value when deleting directories, it would always
+complete a tree delete.  Tasks removing directories will need to explicitly add ``recurse=yes`` if they want to remove
+a directory tree.  When unspecified or no, ``state=absent`` will only remove an empty directory.
```

Comment:
```suggestion
a directory tree.  When unspecified or set to ``no``, ``state=absent`` will only remove an empty directory.
```

Reply:
Looks good, I'll merge it in when I'm back at my desk.

Reply:
Let me know if you like this wording better.